config/manager/manager.yaml

```diff
@@ -16,9 +16,6 @@ spec:
     metadata:
       labels:
     spec:
-      securityContext:
-        runAsNonRoot: true
-        runAsUser: 2000
```

Comment:
Same question here, I thought downgrading the rbac image version took care of the permissions issue, not the case?

Reply:
I updated Makefile so that the securityContext is cleared when the bundle is generated.